test/docker/Test.dockerfile

```diff
@@ -31,6 +32,11 @@ RUN curl -OL https://dl.google.com/dl/cloudsdk/channels/rapid/install_google_clo
     bash install_google_cloud_sdk.bash --disable-prompts --install-dir='/root/' && \
     ln -s /root/google-cloud-sdk/bin/gcloud /usr/local/bin/gcloud
 
+# terraform
+RUN curl -sSL https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip -o /tmp/tf.zip \
```

Comment:
We could use a multistage Docker build and copy it from the official image:
`COPY --from=hashicorp/terraform:0.12.10 <not sure exact path>/terraform /usr/local/bin/terraform`

https://docs.docker.com/develop/develop-images/multistage-build/#use-an-external-image-as-a-stage

Not sure which one would be faster so it probably doesn't matter.

Reply:
I didn't know about this feature, this is awesome 💯. In this case though, it doesn't matter that much. The build time with `curl` was a few seconds faster a couple of times I tried it, but the results were not consistent.